crates/binjs_generate_library/src/lib.rs

```diff
@@ -78,6 +78,9 @@ impl RustExporter {
             TypeSpec::Number =>
                 format!("{prefix}Type::number()",
                     prefix = prefix),
+            TypeSpec::UnsignedLong =>
```

Comment:
I chose `UnsignedLong` to match the WebIDL type "unsigned long", and used it across all parts to avoid confusion (which to use in where).
of course the WebIDL's "usinged long" is U32 tho...
https://heycam.github.io/webidl/#idl-unsigned-long